packages/tuf/src/cjson.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
```

Comment:
I could be useful to add some documentation in this file, like a link to the CJSON spec this implements as well as the reason why we have to do it:
- https://wiki.laptop.org/go/Canonical_JSON
- https://theupdateframework.github.io/specification/v1.0.30/index.html#metaformat

Reply:
Good point! Will do